tests/e2e/transaction/helper.ts

```diff
@@ -39,11 +39,36 @@ export async function publishPackage(
 }
 
 export async function fundAccounts(aptos: Aptos, accounts: Array<Account>) {
-  // eslint-disable-next-line no-restricted-syntax
-  for (const account of accounts) {
-    // eslint-disable-next-line no-await-in-loop
-    await aptos.fundAccount({ accountAddress: account.accountAddress.toString(), amount: FUND_AMOUNT });
-  }
+  // Fund first account
+  const firstAccount = accounts[0];
+  await aptos.fundAccount({ accountAddress: firstAccount.accountAddress.toString(), amount: FUND_AMOUNT });
+  // Fund again because these txns can be expensive
+  await aptos.fundAccount({ accountAddress: firstAccount.accountAddress.toString(), amount: FUND_AMOUNT });
+  const addressesRemaining = accounts.slice(1).map((account) => account.accountAddress);
```

Comment:
Why not just `accounts[1]`

Reply:
I'll put a comment explaining it so it's easier to read, but it's sending the coins from `account[0]` out to `accounts[1..n]` not just `accounts[1]`

Reply:
if `end` in `Array.slice(start?: number, end?: number)` is undefined then it returns the slice from start to the end